datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/glossary/GlossaryTermType.java

```diff
@@ -61,27 +69,24 @@ public EntityType type() {
 
     @Override
     public List<DataFetcherResult<GlossaryTerm>> batchLoad(final List<String> urns, final QueryContext context) {
-        final List<GlossaryTermUrn> glossaryTermUrns = urns.stream()
-                .map(GlossaryTermUtils::getGlossaryTermUrn)
-                .collect(Collectors.toList());
-
+        final Set<Urn> glossaryTermUrns = urns.stream()
```

Comment:
Hi there! This cannot be a set - it must be a list! 

This is because the ordering must be exactly the same as the list of urns passed as an argument